### ISSUE TITLE
Issue with Patcher

### ISSUE THREAD

**Issue by Gug1234** [theSkyseS/Engarde-Synthesis-Patcher#37]
Received this error while running the patcher, was told it was an error within the logic, any idea how to remedy this issue? Thanks.
![error1](https://user-images.githubusercontent.com/101269737/157461291-b69d9991-4be2-4cda-ae85-44fa475df921.PNG)

**Comment by Noggog** [comment]
Looking at their code, this seems to be the section that's problematic:
```
if (_settings.Value.basicAttacks.dwAttackTweaks)
{
    IIdleAnimation idleCopy = CopyIdle(state, Skyrim.IdleAnimation.BlockingStart);
    idleCopy.RelatedIdles[1] = originalLeftHandAttackSibling;
}
```

What this means is that this patcher assumes that Skyrim's Idle Animation `BlockingStart` will have two RelatedIdles.  I'm guessing your setup has some mod that makes it have less?   Would be interesting to see an xEdit screenshot of `BlockingStart`'s related idles section

Either way, I'm not sure if the idle having 2 entries is a fundamental requirement of Engarde, or if it's an oversight by the author.  I'm not experienced enough with the goals or mechanics of the patcher to know offhand

**Comment by theSkyseS** [comment]
Should be fixed with 6008d66. Feel free to reopen the issue if still wont work.
You should probably check which mod changes the BlockingStart idle, it may or may not work with the dual wield part of Engarde
